<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<testResult type="org.overture.codegen.tests.exec.TracesVerdictJavaGenTest">
<result>
<output object="rO0ABXNyABBqYXZhLnV0aWwuVmVjdG9y2Zd9W4A7rwEDAANJABFjYXBhY2l0eUluY3JlbWVudEkADGVsZW1lbnRDb3VudFsAC2VsZW1lbnREYXRhdAATW0xqYXZhL2xhbmcvT2JqZWN0O3hwAAAAAAAAAAR1cgATW0xqYXZhLmxhbmcuT2JqZWN0O5DOWJ8QcylsAgAAeHAAAAAKc3IAKW9yZy5vdmVydHVyZS5jdC5jdHJ1bnRpbWUudXRpbHMuVHJhY2VUZXN0feFkkkPBXUACAARMAAJub3QAE0xqYXZhL2xhbmcvSW50ZWdlcjtMAAZyZXN1bHR0ABJMamF2YS9sYW5nL1N0cmluZztMAAR0ZXN0cQB+AAdMAAd2ZXJkaWN0dAApTG9yZy9vdmVydHVyZS9pbnRlcnByZXRlci90cmFjZXMvVmVyZGljdDt4cHNyABFqYXZhLmxhbmcuSW50ZWdlchLioKT3gYc4AgABSQAFdmFsdWV4cgAQamF2YS5sYW5nLk51bWJlcoaslR0LlOCLAgAAeHAAAAABdAA4RXJyb3IgNDA1ODogVmFsdWUgMCBpcyBub3QgYSBuYXQxIGluICdFbnRyeScgYXQgbGluZSA5OjN0AARmKDApfnIAJ29yZy5vdmVydHVyZS5pbnRlcnByZXRlci50cmFjZXMuVmVyZGljdAAAAAAAAAAAEgAAeHIADmphdmEubGFuZy5FbnVtAAAAAAAAAAASAAB4cHQABkZBSUxFRHNxAH4ABXNxAH4ACgAAAAJ0AAExdAAEZigxKX5xAH4AD3QABlBBU1NFRHNxAH4ABXNxAH4ACgAAAAN0AAEydAAEZigyKXEAfgAXc3EAfgAFc3EAfgAKAAAABHQAQUVycm9yIDQwODM6IFNlcXVlbmNlIGluZGV4IG91dCBvZiByYW5nZTogMyBpbiAnRW50cnknIGF0IGxpbmUgOTozdAAEZigzKXEAfgARcHBwcHBweA==" resource="SeqIndexOutOfRange.vdmpp" value="[TraceTest: Number: 1 Test: f(0) Result: Error 4058: Value 0 is not a nat1 in 'Entry' at line 9:3 Verdict: FAILED, TraceTest: Number: 2 Test: f(1) Result: 1 Verdict: PASSED, TraceTest: Number: 3 Test: f(2) Result: 2 Verdict: PASSED, TraceTest: Number: 4 Test: f(3) Result: Error 4083: Sequence index out of range: 3 in 'Entry' at line 9:3 Verdict: FAILED]"/>
</result>
</testResult>
